### ISSUE TITLE
wish: pages/document size

### ISSUE THREAD

**Issue by furter** [hybrid-publishing-lab/publishing-converter#1]
Could we have the pages/document size translated somewhere in the CSS? 
And then, how could it be used/adapted?

**Comment by gimsieke** [comment]
You mean, the page count? Which size do you mean? idml or html file size in bytes? Or just the page count?
The page count is not something that can be extracted from the IDML because it may only be calculated when the document is layouted. 
We need to adapt the [page name insertion script](https://github.com/le-tex/InDesignScripts/tree/master/pagenames) to InDesign files without a “main” story before we can more or less reliably insert the page names (= numbers, in most cases) into the text prior to exporting IDML. Then we may calculate a page count, but still not a reliable one. 
What do you need it for?

**Comment by furter** [comment]
I mean the dimensions of the page (the dimensions of the book), it could be useful to have a sense of scale.

**Comment by gimsieke** [comment]
Yes, this should be feasible. There may be different page/spread dimensions for each (master) spread though, but I think we can ignore that. We can select the first spread and convert its dimensions to a `@page` at rule in CSS.
